server/ServerApp.cpp

```diff
@@ -892,7 +900,8 @@ void ServerApp::SendNewGameStartMessages() {
                                                         m_universe,              m_species_manager,
                                                         GetCombatLogManager(),   m_supply_manager,
                                                         player_info_map,         m_galaxy_setup_data,
-                                                        use_binary_serialization,!player_connection->IsLocalConnection()));
+                                                        use_binary_serialization,!player_connection->IsLocalConnection()),
+                                       boost::bind(&ServerApp::UpdateEmpireTurnReceive, this, empire_id, m_current_turn, boost::placeholders::_1));
```

Comment:
could probably safely use a lambda here rather than `boost::bind`

Reply:
Changed it.